### ISSUE TITLE
Test Explorer show no tests.

### ISSUE THREAD

**Issue by Deedolith** [rubberduck-vba/Rubberduck#5917]
**Rubberduck version information**
The info below can be copy-paste-completed from the first lines of Rubberduck's log or the About box:

Version 2.5.2.6027
OS: Microsoft Windows NT 10.0.19043.0, x64
Host Product: Microsoft Office 2013 x64
Host Version: 15.0.5349.1000
Host Executable: MSACCESS.EXE

**Description**
Within the Test Explorer window, after adding a Test Module, nothing new show up in the Test Explorer window.
The "ducks" loading trails keep showing forever.

**To Reproduce**
1. Go to the RubberDuck menu
2. Click on RubberDuck ==> Unit Tests ==> Test Explorer
3. Go to the Test Explorer window
4. Click on Add ==> Test Module
5. Go to The Code Explorer window
6. Double click on the newly created TestModule1
7. Go to the Test Explorer window
8. Click on Add ==> Test Method

**Expected behavior**
A new line holding tests informations should show up in the Test Window

**Screenshots**
![image](https://user-images.githubusercontent.com/1517410/145718024-4f1daf92-d8cd-4584-b196-6802946e8b48.png)

**Logfile**
[RubberduckLog.txt](https://github.com/rubberduck-vba/Rubberduck/files/7698984/RubberduckLog.txt)

**Additional context**
Behavior observer within both MS Access and MS Excel (2013).

**Comment by BZngr** [comment]
FWIW - I could not reproduce this issue following the steps described above (Excel host).   It behaved as intended, so ATM this appears to be an intermittent failure. 

From the graphic, it looks like the issue was observed on the first `TestMethod` added to the first `TestModule`...so I tried that scenario, again with success.  And, finally...I went thru the steps when the project was in a 'non-compilable' state.  Still worked as expected.
